tmail-backend/tmail-third-party/open-ai/AIBotMailet.adoc

```diff
@@ -0,0 +1,23 @@
+= AIBotMailet
+
+`AIBotMailet` is a mailet designed to automatically reply to emails sent to a specific recipient (e.g., gpt@linagora.com). It uses the OpenAI API by default but can be configured to use any LLM API supported by https://github.com/langchain4j/[langchain4j].
+
+== Configuration Example
+
+[source,xml]
+----
+<mailet match="com.linagora.tmail.mailet.RecipientsContain=gpt@linagora.com" class="com.linagora.tmail.mailet.AIBotMailet">
+    <apiKey>demo</apiKey>
+    <gptAddress>gpt@tmail.com</gptAddress>
+    <model>openai,gpt-4o-mini</model>
+</mailet>
```

Comment:
baseUrl missing in the example

Reply:
The need for a base URL is the special case here, normally, langchain4j (the library we use to interact with the different LLMs APIs) would know which base URL to use by default, for the different LLMs it supports (OpenAI, Mistral, Llama, etc).

However, the library does not support ai.linagora.com. Fortunately, AI Liangora is compatible with the OpenAPI API, which enables us to use the OpenAI API implementation in langchain4j and change the baseURL.

Reply:
Fair, thanks for the explanation. Shouldn't that be precised in the doc though then?

Reply:
Makes sense, documentation updated.